test/src/main_test.cpp

```diff
@@ -24,42 +24,39 @@
 
 #include <csignal>
 #include <spdlog/spdlog.h>
-#include <up-client-zenoh-cpp/transport/zenohUTransport.h>
+#include <up-client-zenoh-cpp/client/upZenohClient.h>
 #include <gtest/gtest.h>
 
 using namespace uprotocol::utransport;
 
 class upClientZenohCppTest : public ::testing::Test {
 
     public:
+        static std::shared_ptr<upZenohClient> pClient;
```

Comment:
Change to `static inline ...` to avoid an `undefined reference to` linker error

Reply:
That did it.  Thanks!